### DISCUSSION TITLE
Someting went wrong, when I'm using Adguard and Clash together

### DISCUSSION THREAD

**Discussion by LawyZheng** [AdguardTeam/AdGuardHome#5920] [Q&A]
My mac mini has already installed AdGuard And ClashX Pro.

Recently, if I enabled ClashX Pro, I found my browser injection endpoint turned into `injections.adguard.org` from `local.adguard.org`  when I use Chrome to browse websites.

And I found that requests to this domain didn't return any data until err happened or timed out (about 60s to time out). So, it slows me down to browse the websites.

What should I do to fix this problem?

Thanks a lot.

**Comment by as232** [comment]
Over 3 years later and no response... not good...

**Reply by Dynamic5912** [reply]
It's a discussion section, not an issue or bug report section...

Do you have some input?
